apps/ledger-live-mobile/src/logic/postOnboarding/hooks.tsx

```diff
@@ -0,0 +1,162 @@
+import { useDispatch, useSelector } from "react-redux";
+import { useCallback, useMemo } from "react";
+import { useNavigation } from "@react-navigation/native";
+import { DeviceModelId } from "@ledgerhq/devices/lib/index";
+import {
+  PostOnboardingActionId,
+  PostOnboardingHubState,
+} from "@ledgerhq/types-live";
+import { useFeatureFlags } from "@ledgerhq/live-common/lib/featureFlags/index";
+import { getPostOnboardingAction, getPostOnboardingActionsForDevice } from ".";
+import {
+  hubStateSelector,
+  postOnboardingDeviceModelIdSelector,
+  walletPostOnboardingEntryPointDismissedSelector,
+} from "../../reducers/postOnboarding";
+import {
+  clearPostOnboardingLastActionCompleted,
+  hidePostOnboardingWalletEntryPoint,
+  initPostOnboarding,
+  setPostOnboardingActionCompleted,
+} from "../../actions/postOnboarding";
+import { NavigatorName, ScreenName } from "../../const";
+
+/**
+ * @returns an object representing the state that should be rendered on the post
+ * onboarding hub screen.
+ *
+ * This takes feature flagging into account so the logic is
+ * resistant to flags getting enabled/disabled over time (for a given disabled
+ * feature flag, the actions pointing to it will be excluded).
+ * */
+export function usePostOnboardingHubState(): PostOnboardingHubState {
```

Comment:
Can you add some tests like @alexandremgo to cover this hook.
We agreed that this was the right way to go even if it's a bit of a pita 🙏🏼

Reply:
This _and other_ hooks

Reply:
👍 I will do that